### PR TITLE
Updated README.md on lung datasets

### PR DESCRIPTION
**Pull request by pranshurastogi29**

### CODE REVIEW

**Comment by ucalyptus** [comment]
reference the issue link here in the comments. do this for every pr u make

File: README.md [hunk 1]
```diff
@@ -48,3 +48,7 @@ https://colab.research.google.com/drive/1iT79EPridQifgGxkZhh_YpuKq5uRflhu
    https://medium.com/@move37timm/using-kaggle-api-for-google-colaboratory-d18645f93648
 2. if you does not have a account and still wants to run the notebook
    you can use the my kaggle.json in the repo
+
+### some links for lung related tasks
+https://grand-challenge.org/challenges/
```
Comment (ucalyptus) [inline README.md]: can u mention datasets and  give links to download?

**Review by ucalyptus — CHANGES_REQUESTED** [review]

**Comment by ucalyptus** [comment]
another update: there was no plans to highlight this on readme. we just want to keep a separate markdown for this.

File: README.md (hunk 1 above)
Comment (pranshurastogi29) [reply README.md]: we cannot download the dataset straight forward we have to participate and then we can down load the data i have provided the links for where to find the lungs dataset

**Comment by pranshurastogi29** [comment]
seperate markdown?

**Comment by ucalyptus** [comment]
a dataset.md

**Pull request closed without merging**